src/test/java/uk/gov/dhsc/htbhf/page/EnterDobPage.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.dhsc.htbhf.page;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import uk.gov.dhsc.htbhf.page.component.InputField;
+import uk.gov.dhsc.htbhf.page.component.SubmitButton;
+
+/**
+ * Page object for the Enter Date of Birth page.
+ */
+public class EnterDobPage extends BasePage {
+
+    private InputField dayInputField;
+    private InputField monthInputField;
+    private InputField yearInputField;
+    private SubmitButton submitButton;
+
+    public EnterDobPage(WebDriver webDriver, String baseUrl, WebDriverWait wait) {
+        super(webDriver, baseUrl, wait);
+        this.dayInputField = new InputField(webDriver, "dateOfBirth-day");
+        this.monthInputField = new InputField(webDriver, "dateOfBirth-month");
+        this.yearInputField = new InputField(webDriver, "dateOfBirth-year");
+        this.submitButton = new SubmitButton(webDriver);
+    }
+
+    @Override
+    String getPath() {
+        return "/enter-dob";
+    }
+
+    @Override
+    String getPageName() {
+        return "enter date of borth";
```

Comment:
Typo `borth` -> `birth`

Reply:
Yep, will correct, it's not used until I implement the navigation tests so hadn't spotted that yet.